pkg/cluster/cluster.go

```diff
@@ -464,11 +472,24 @@ func (c *Cluster) pollPods() ([]*v1.Pod, []*v1.Pod, error) {
 		case v1.PodRunning:
 			running = append(running, pod)
 		case v1.PodPending:
-			pending = append(pending, pod)
+			unready = append(unready, pod)
+		}
+	}
+
+	for _, pod := range running {
+		url := fmt.Sprintf("http://%s:2379", pod.Status.PodIP)
```

Comment:
We probably should pass URL struct instead of string for further TLS integration.
Probably add a todo?